service_files/istep_services/istep-alignment_check@.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=istep alignment_check for host %i
+After=istep-updatehwmodel@%i.service
+Requires=istep-updatehwmodel@%i.service
+ConditionPathExists=!/run/openbmc/host@%i-on
+Conflicts=obmc-host-stop@%i.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/istep alignment_check
+RemainAfterExit=yes
```

Comment:
do we need to remain after exit, i mean this service will be still active even after the end of the istep, is that required.